axlearn/cloud/gcp/project_setup.sh

```diff
@@ -0,0 +1,71 @@
+# This is a script to set up a brand new GCP project before you use AXLearn tools"""
+#
+# Usage: 
+#    # fill out environment variables below
+#    chmod +x project_setup.sh
+#    ./project_setup.sh
+
+
+#!/bin/sh
+
+set -e
+#set -x
```

Comment:
Is this commented line intentional?

Reply:
I figured you can test both and see whether you want to include this line. From my own experience, including "set -x" made my terminal output confusing...

Reply:
If it's not adding much benefits we can remove it.

Reply:
cool, edited